Detectors/ITSMFT/ITS/macros/test/CheckClusters.C

```diff
@@ -37,7 +37,7 @@ void CheckClusters(std::string clusfile = "o2clus_its.root", std::string hitfile
   std::vector<MC2HITS_map> mc2hitVec;
 
   TFile fout("CheckClusters.root", "recreate");
-  TNtuple nt("ntc", "cluster ntuple", "x:y:z:dx:dz:lab:ev:rof:hlx:hlz:clx:clz:n:id");
+  TNtuple nt("ntc", "cluster ntuple", "x:y:z:dx:dz:lab:ev:hlx:hlz:clx:clz:n:id");
```

Comment:
I think it is useful to have the ROFrame in the output, but instead of extracting it from the cluster, one should use  ``rofRec.getROFrame()``

Reply:
@shahor02 Indeed, true.  I'll fix it asap...